hedwiki/score/HED_score_0.0.1.mediawiki

```diff
@@ -508,23 +508,23 @@ Reference:
             *** High-range <nowiki>[High ( more than 70 microV)]</nowiki>
             **** PDR-amplitude-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
 
-        ** PDR-Amplitude-asymmetry <nowiki>[A difference in amplitude between the homologous area on opposite sides of the head that consistently exceeds 50 percent. When symmetrical could be labeled with base schema 'Symmetrical' tag]</nowiki>
+        ** PDR-Amplitude-asymmetry <nowiki>[A difference in amplitude between the homologous area on opposite sides of the head that consistently exceeds 50 percent. When symmetrical could be labeled with base schema Symmetrical tag]</nowiki>
```

Comment:
In this section:   `PDR-Amplitude`... should be `PDR-amplitude`.

Also suggest changing items  like `LowerLeft` to `lower-left` in this section.